docs/en/sql-reference/10-sql-commands/50-administration-cmds/02-unset.md

```diff
@@ -3,29 +3,40 @@ title: UNSET
 ---
 import FunctionDescription from '@site/src/components/FunctionDescription';
 
-<FunctionDescription description="Introduced or updated: v1.2.467"/>
+<FunctionDescription description="Introduced or updated: v1.2.605"/>
 
-Reverts one or more system settings to their default levels and values. For more information about the setting levels, see [Setting Levels](03-show-settings.md#setting-levels). To show all the current settings, use [SHOW SETTINGS](03-show-settings.md).
+Reverts one or more system settings to their global or default levels and values. For more information about the setting levels, see [Setting Levels](03-show-settings.md#setting-levels). To show all the current settings, use [SHOW SETTINGS](03-show-settings.md).
 
 See also: [SET](02-set-global.md)
 
 ## Syntax
 
 ```sql
 -- Unset one setting
-UNSET [ SESSION ] <setting_name> 
+UNSET [ SESSION | GLOBAL ] <setting_name> 
 
 -- Unset multiple settings
-UNSET [ SESSION ] ( <setting_name>, <setting_name> ... )
+UNSET [ SESSION | GLOBAL ] ( <setting_name>, <setting_name> ... )
 ```
 
-| Parameter | Description                                                                                                                                               |
-|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
-| SESSION   | Removes the session-level (current session) setting of a global-level setting, reverting the setting back to the global level and its global-level value. |
+| Parameter | Description                                                                                                                                                                                                                                                                                                             |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SESSION   | If setting has global-level value, removes the session-level (current session) setting of a global-level setting, reverting the setting back to the global level and its global-level value.<br/>If settings just has session-level value, reverting the setting back to the DEFAULT level and its default-level value. |
```

Comment:
If the setting has a global-level value, it removes the session-level override and reverts to the global setting. If the setting only has a session-level value, it reverts to the default setting.